test/mind_the_gapp/arrival_time_test.exs

```diff
@@ -0,0 +1,75 @@
+defmodule MindTheGapp.ArrivalTimeTest do
+  use MindTheGapp.DataCase, async: true
+  alias MindTheGapp.Mta.ArrivalTime
+
+  setup_all :create_fake_arrival_time_data
+
+  def create_fake_arrival_time_data(_context) do
+    clear_database()
+    generate_fake_arrival_data()
+    :ok
+  end
+
+  def clear_database() do
+    MindTheGapp.Repo.delete_all(MindTheGapp.ArrivalTime)
+  end
+
+  def generate_fake_arrival_data() do
+    starting_date = DateTime.utc_now
+
+     1..50
+     |> Enum.reduce(starting_date, &persist_arrival_time/2)
+  end
+
+  def persist_arrival_time(_x, date_time) do
+    actual_time = DateTime.add(date_time, Enum.random(-10..10) * 60)
```

Comment:
Do we have a test to ensure the query is taking the average of the arrival times?

Reply:
We do not; I will add that